airflow/migrations/versions/0050_3_0_0_remove_pickled_data_from_xcom_table.py

```diff
@@ -136,6 +136,7 @@ def upgrade():
         # Drop the old `value_old` column
         with op.batch_alter_table("xcom", schema=None) as batch_op:
             batch_op.drop_column("value_old")
+    op.drop_table("_xcom_archive")
```

Comment:
I didn't drop it purposely to avoid loss of data, was planning to handle it as part of `db cleanup`

Reply:
A little fix here: https://github.com/apache/airflow/pull/44221/commits/47128e97fcc6614168c023a3f15db4c38a23eaf5